clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -11341,6 +11341,23 @@ void LinkerWrapper::ConstructJob(Compilation &C, const JobAction &JA,
       CmdArgs.push_back(
           Args.MakeArgString("--sycl-post-link-options=" + PostLinkOptString));
 
+    if (Args.hasArg(options::OPT_fsycl_remove_unused_external_funcs))
```

Comment:
I noticed that the tests for these options pass `--offload-new-driver`, but there doesn't seem to be a corresponding check for `--offload-new-driver` in the code before these options are added to the command.
Could you please clarify this

Reply:
This is in `LinkerWrapper::ConstructJob`, which I believe should only be called in the new offload driver. Is this ok?